crowns.lic

```diff
@@ -117,27 +117,25 @@ class Dice
   end
 
   def check_prize
-    if @trash_items.any? { |item| Regexp.new(item) =~ right_hand }
-      bput("put my #{righthand} in bucket", 'You put', 'You drop')
-    else
-      case bput("stow #{righthand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
-      when 'to fit in the'
-        message("*** Item is too big to fit in your stow container! ***")
-        exit
-      end
-      pause
-    end
-    # Check left hand as well. On lag may have items in both hands.
-    if @trash_items.any? { |item| Regexp.new(item) =~ left_hand }
-      bput("put my #{lefthand} in bucket", 'You put', 'You drop')
-    else
-      case bput("stow #{lefthand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
-      when 'to fit in the'
-        message("*** Item is too big to fit in your stow container! ***")
-        exit
+    [left_hand, right_hand]
+      .compact
+      .each do |in_hand|
+        if @trash_items.any? { |item| Regexp.new(item) =~ in_hand }
+          bput("put my #{in_hand} in bucket", 'You put', 'You drop')
+        else
+          # Coil rope so we can store it
+          if /\brope\b/ =~ in_hand
+            fput("coil my #{in_hand}")
+          end
+          case bput("stow #{in_hand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
+          when 'to fit in the'
```

Comment:
This should have caught the rope was too big.

Reply:
It did catch that the rope was too long but the script aborted unecessarily because it was a recoverable error. That's why I added the `coil rope` step above so that it could be stored. You pointed out that the junk filter wasn't working always and would have filtered the rope out to begin with, but that didn't happen for me so I added the behavior to handle it just in case.

Reply:
No worries. I have the junk list working better but I like the catch. Who knows, the person may want the rope. :)